whobpyt/models/RWW/wong_wang.py

```diff
@@ -144,42 +126,16 @@ def __init__(self, node_size: int,
         self.steps_per_TR = int(tr / step_size)
         self.TRs_per_window = TRs_per_window  # size of the batch used at each step
         self.node_size = node_size  # num of ROI
-        self.sampling_size = sampling_size
         self.sc = sc  # matrix node_size x node_size structure connectivity
         self.sc_fitted = torch.tensor(sc, dtype=torch.float32)  # placeholder
         self.use_fit_gains = use_fit_gains  # flag for fitting gains
-        self.use_Laplacian = use_Laplacian
-        self.use_Bifurcation = use_Bifurcation
```

Comment:
Same as #174, #175 is a to-do issue flagging that we need to put the bifurcation analysis component back into the code. Minimally, because this is used in the biorxiv paper draft that needs to be completed and submitted and reproducible within `whobpyt`.

Reply:
@Andrew-Clappison unless you have any objections I will go ahead and merge

Reply:
Glad to see a trend towards simplification of the model and objective function. Didn’t look too closely, but here are some additional non-critical things I noticed that can be considered later:

-	Argument type checking has been removed
-	The RWWEI model is no longer a subclass of AbstractNMM

Reply:
Ok thanks @Andrew-Clappison for the additional notes. We will refer to this discussion thread moving forward as we get to these points. 

Going ahead with merge now.